railties/test/commands/credentials_test.rb

```diff
@@ -55,6 +55,15 @@ class Rails::Command::CredentialsCommandTest < ActiveSupport::TestCase
     end
   end
 
+  test "edit_command_does_not_add_master_key_when config/credentials.yml.enc exists" do
```

Comment:
If using the `test` method, you do not need to use underscores to delimit words. Please use spaces instead.